### PR TITLE
"Temporary test to assert 401 status"

### PR DESCRIPTION
**Pull request by LinaAbu1306**

### CODE REVIEW

File: tests/api_tests/getUser.spec.ts [hunk 1]
```diff
@@ -5,3 +5,10 @@ test.skip('should check response status', { tag: ['@api'] }, async ({ request })
     console.log(JSON.stringify(response));
     expect(response['_initializer'].status).toBe(200);
 });
+test('Assert 401 response status', async ({request}) =>{
```
Comment (Elgrueber) [inline tests/api_tests/getUser.spec.ts]: I would change the name of the test. You write the negative test for getting of list of users?

File: tests/api_tests/getUser.spec.ts [hunk 2]
```diff
@@ -5,3 +5,10 @@ test.skip('should check response status', { tag: ['@api'] }, async ({ request })
...
+    const response = await request.get (process.env.API_BASE_URL + '/user')
+    expect (response.status()).toBe(401)
+    
+    const responseBody = JSON.parse (await response.text())
+    console.log(responseBody)
```
Comment (Elgrueber) [inline tests/api_tests/getUser.spec.ts]: please remove console.log. If you want to add verify - better to add it with expect using message in the responseBody

Comment (Elgrueber) [reply tests/api_tests/getUser.spec.ts]: message or text

File: tests/api_tests/getUser.spec.ts (hunk 1 above)
Comment (LinaAbu1306) [reply tests/api_tests/getUser.spec.ts]: done,
please review

File: tests/api_tests/getUser.spec.ts [hunk 3]
```diff
@@ -5,3 +5,9 @@ test.skip('should check response status', { tag: ['@api'] }, async ({ request })
     console.log(JSON.stringify(response));
     expect(response['_initializer'].status).toBe(200);
 });
+test('Verify 401 Unauthorized response status', async ({request}) =>{
+    const response = await request.get (process.env.API_BASE_URL + '/user')
```
Comment (Elgrueber) [inline tests/api_tests/getUser.spec.ts]: in the file apidata there I created end-points. you can take from there

File: tests/api_tests/getUser.spec.ts [hunk 4]
```diff
@@ -5,3 +5,9 @@ test.skip('should check response status', { tag: ['@api'] }, async ({ request })
     console.log(JSON.stringify(response));
     expect(response['_initializer'].status).toBe(200);
 });
+test('Verify 401 Unauthorized response status', async ({request}) =>{
```
Comment (Elgrueber) [inline tests/api_tests/getUser.spec.ts]: better to change name for the name what you verify (from swagger)

File: tests/api_tests/getUser.spec.ts [hunk 5]
```diff
@@ -1,7 +1,15 @@
 import { test, expect } from '@playwright/test';
+import { API_URL_END_POINTS } from '../../apiData';
 
 test.skip('should check response status', { tag: ['@api'] }, async ({ request }) => {
     const response = await request.get(process.env.API_BASE_URL + '/user');
     console.log(JSON.stringify(response));
     expect(response['_initializer'].status).toBe(200);
 });
+test('Negative test| Unauthorized user by GET api/user', async ({ request }) => {
```
Comment (Elgrueber) [inline tests/api_tests/getUser.spec.ts]: please add "to retrieve all users"

File: tests/api_tests/getUser.spec.ts (hunk 1 above)
Comment (Elgrueber) [reply tests/api_tests/getUser.spec.ts]: Please improve test name (make it focusing on what is being tested)

**Review by IQA-App — APPROVED** [review]

**Pull request merged**